invenio_communities/static/templates/invenio_communities/ng_record_curate.html

```diff
@@ -1,27 +1,44 @@
-<div class="row record-elem" ng-repeat="record in vm.invenioSearchResults.hits.hits track by $index">
+<div id="communities-rec-{{ record.id }}" class="row record-elem" ng-repeat="record in vm.invenioSearchResults.hits.hits track by $index">
   <div class="col-xs-12">
     <div class="pull-right view-record hidden-xs">
       <span class="pull-right" id="curate_{{ record.id }}">
         <span class="ajaxmsg text-muted" style="display: none;"></span>
-        <a class="btn 10 btn-default" ng-class="pressedClassSucc" ng-click="handleClick('accept', record.id);isPressed=true;pressedClassSucc='btn-success'" ng-disabled="isPressed"><i class="fa fa-check"></i>Accept</a>
-        <a class="btn 10 btn-default" ng-class="pressedClassWarn" ng-click="handleClick('reject', record.id);isPressed=true;pressedClassWarn='btn-danger'" ng-disabled="isPressed"><i class="fa fa-check"></i>Reject</a>
+        <a class="btn 10 btn-default" ng-class="pressedClassSucc" ng-click="CommunitieshandleClick('accept', record);isPressed=true;pressedClassSucc='btn-success'" ng-disabled="isPressed"><i class="fa fa-check"></i>Accept</a>
+        <a class="btn 10 btn-default" ng-class="pressedClassWarn" ng-click="CommunitieshandleClick('reject', record);isPressed=true;pressedClassWarn='btn-danger'" ng-disabled="isPressed"><i class="fa fa-check"></i>Reject</a>
```

Comment:
this is a bug that should be fixed, otherwise it leads to silent errors because the theme already has a function "handleClick", at least in zenodo, see https://github.com/zenodo/zenodo/search?utf8=%E2%9C%93&q=handleClick